apps/studio/src/pages/home/owners/OwnersTable.tsx

```diff
@@ -212,7 +212,7 @@ export default function OwnersTable({
                       `${firstName} ${lastName}`
                     ) : (
                       <Typography fontStyle="italic" fontWeight={ 400 }>
```

Comment:
This italics is a bit strange and seems out of place to other font styles we've used... can't find any reference to it on Figma either.